x/mongo/driver/topology/pool_test.go

```diff
@@ -15,6 +15,7 @@ import (
 	"time"
 
 	"github.com/stretchr/testify/assert"
+
```

Comment:
```suggestion
```

Reply:
[c474832](https://github.com/mongodb/mongo-go-driver/pull/1058/commits/c4748327cd33f4035beaaac427511bcbb2015a0a)

fix it. My IDE setting inserts new line. I wish there was a mention in `CONTRIBUTING.md` for like a this case.

Reply:
Ah I assume your IDE is set up to separate external and internal imports with a whitespace. We do recommend in `CONTRIBUTING.md` that contributors run `make fmt`, which should get rid of issues like this.